Misc/NEWS.d/next/Library/2024-06-15-23-38-36.gh-issue-120284.HwsAtY.rst

```diff
@@ -0,0 +1 @@
+Make asyncio.Runner's run() method accept the same object types as asyncio.run_until_complete(). Instead of accepting only a coroutine, run() will now accept an awaitable, coroutine, asyncio.Future, or asyncio.Task.
```

Comment:
use rst markup please

Reply:
Done, though I wasn't sure how to validate that the references will work.